src/EnergyPlus/AirflowNetwork/src/Solver.cpp

```diff
@@ -197,7 +191,7 @@ namespace AirflowNetwork {
         if (simulation_control.type == ControlType::NoMultizoneOrDistribution) return;
 
         if (m_state.dataGlobal->BeginEnvrnFlag) {
-            TurnFansOn = false; // The FAN should be off when BeginEnvrnFlag = .True.
+            m_state.dataHVACGlobal->TurnFansOn = false; // The FAN should be off when BeginEnvrnFlag = .True.
```

Comment:
👍

Reply:
Separate question. I notice that in some places, AFN uses `state` parameters and in others it uses an `m_state` member. Why? I have not traced where `m_state` is set, but I have a sinking suspicion that it may be making EnergyPlus thread unsafe. Has anyone looked into this?

Reply:
I am using `m_state` in a few places to get `state` out of the interface. It is a temporary measure as I continue to refactor things. I do not believe it is making EnergyPlus thread unsafe.